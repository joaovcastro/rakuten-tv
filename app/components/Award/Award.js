import React from 'react';
import PropTypes from 'prop-types';

const Award = ({ name, category, result }) => (
  <div className="flex center line-height-5">
    <div className="award__laurea--left" />
    <div className="center">
      <p className="bold">{category}</p>
      <p className="bold golden">{name}</p>
      <p>{result}</p>
    </div>
    <div className="award__laurea--right" />
  </div>
);

Award.propTypes = {
  name: PropTypes.string,
  category: PropTypes.string,
  result: PropTypes.string,
};

export default Award;
