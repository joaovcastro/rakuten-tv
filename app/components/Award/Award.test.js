import React from 'react';
import { shallow } from 'enzyme';

import Award from './Award';

describe('<Award />', () => {
  it('correctly renders', () => {
    const wrapper = shallow(<Award />);
    expect(wrapper).toMatchSnapshot();
  });
});
