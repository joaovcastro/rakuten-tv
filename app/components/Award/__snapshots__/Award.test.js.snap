// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Award /> correctly renders 1`] = `
<div
  className="flex center line-height-5"
>
  <div
    className="award__laurea--left"
  />
  <div
    className="center"
  >
    <p
      className="bold"
    />
    <p
      className="bold golden"
    />
    <p />
  </div>
  <div
    className="award__laurea--right"
  />
</div>
`;
